RadixWallet/Features/DappInteractionFeature/Coordinator/DappInteractionFlow.swift

```diff
@@ -472,17 +472,27 @@ extension DappInteractionFlow {
 
 		func handleAccountsProofOfOwnership(
 			_ item: State.AnyInteractionItem,
-			_ accountAuthProofs: [AccountAuthProof],
-			_ signedAuthChallenge: SignedAuthChallenge
+			_ accountAddresses: [AccountAddress],
+			_ signedAuthIntent: SignedAuthIntent
 		) -> Effect<Action> {
-			let proofs = accountAuthProofs.map {
-				WalletToDappInteractionProofOfOwnership.account(.init(
-					accountAddress: $0.account.address,
-					proof: $0.proof
-				))
+			// Build account proofs
+			let accountProofs: [WalletToDappInteractionAccountProof] = signedAuthIntent.intentSignaturesPerOwner.compactMap { item in
```

Comment:
I think this can be:

```swift
let accountProofs = signedAuthIntent.intentSignaturesPerOwner.compactMap { item in
	item.owner.accountAddress.map {
		WalletToDappInteractionAccountProof(
			accountAddress: $0,
			proof: .init(intentSignatureOfOwner: item)
		)
	}
}
```

which I find more readable. but matter of taste